Objects/unionobject.c

```diff
@@ -422,6 +422,30 @@ static PyNumberMethods union_as_number = {
         .nb_or = _Py_union_type_or, // Add __or__ function
 };
 
+static const char* const cls_attrs[] = {
+        "__name__",
+        "__qualname__",
+        "__module__",
```

Comment:
Please add the comment ``Required for compatibility with typing module``. I would be confused if I saw this in other places 😉 .

Reply:
Yup, thanks for pointing on this)